client/web-sveltekit/src/lib/dom.ts

```diff
@@ -247,13 +247,15 @@ interface ComputeFitAttributes {
 export const computeFit: Action<HTMLElement, void, ComputeFitAttributes> = node => {
     // Holds the cumulative width of all elements up to element i.
     const widths: number[] = [0]
+    const parentRight = node.getBoundingClientRect().left
```

Comment:
Despite what I side during our call, this is not actually the right side of the parent, it's the left 😅 
I suggest we rename this do something more neutral, `offset` and add a comment explaining the intention behind it.

```suggestion
    // Used to compute the "end" of each child relative to the parent container.
    // This ensures that the logic here still works when the parent node is moved
    // or when it is scrolled inside a scroll container.
    const offset = node.getBoundingClientRect().left
```